### DISCUSSION TITLE
Are there standards for orientation of a splat?

### DISCUSSION THREAD

**Discussion by rowild** [mkkellogg/GaussianSplats3D#348] [Q&A]
First of all I would like to express my gratitude and fascination with this project! Simply amazing! What a pity that a day only has 24 hours... :-)

I get my splats mainly from Lumalabs (drone flights around old castles). Then I pull the splats into LumaLabs to do some cleanup work. And eventually I view them in GaussianSplats3D ... where they are upside down. This is what puzzles me a bit, because what Lumalabs (or Polycam, KiriEngine, etc) export, is shown properly SuperSplat (and also in Blender, when it comes to point clouds). 

May I ask what is the decision being this? I am not familiar with other 3D software like Maya or UnRealEngine – does GaussianSplats3d adopt their orientation standards? 

What would be the easiest way to prepare a model for GaussianSplats3d? Currently I export my models from SuperSplat 180 degree turn around in z-axis. But of course it would be great, if there was some kind of standard. Honestly, to me it seems this might be possibly a bug?

I uploaded 3 images that hopefully better explain what I try to say... Thank you!

![1_lumalabs](https://github.com/user-attachments/assets/27dc9858-c23b-43d9-a98f-1f828af32c9c)
![2_supersplat](https://github.com/user-attachments/assets/073c5b58-3bc4-49b9-a0a0-9afb6a2afa27)
![3_gaussiansplats3d](https://github.com/user-attachments/assets/82fa613e-5159-4a32-92d4-53629d18b191)

**Comment by rowild** [comment]
Anybody?

**Reply by mkkellogg** [reply]
Another developer encountered a similar issue a while back, and at the time I discovered Super Splat had some additional logic to invert the scene when it imports it. The issue can be found here: https://github.com/mkkellogg/GaussianSplats3D/issues/186. Are you exporting your `.ply` compressed or uncompressed?
